PWGHF/TableProducer/candidateSelectorLc.cxx

```diff
@@ -101,6 +101,8 @@ struct HfCandidateSelectorLc {
 
   HistogramRegistry registry{"registry"};
 
+  double massK0Star892 = 0.89555; // M(K*0(892)) = 895.55 Mev/c2
```

Comment:
I added the K* mass to the physics constants in O2, let's remember to modify this hard-coded value once https://github.com/AliceO2Group/AliceO2/pull/13235 gets merged

Reply:
good to know, thank you